docker-compose.e2e.yml

```diff
@@ -0,0 +1,25 @@
+version: "3.7"
+services:
+  collector:
+    build:
+      context: .
+      dockerfile: ./e2e-docker/Dockerfile.tool
+    ports:
+      - 19876:19876
+      - 12800:12800
+
+  consumer:
+    build:
+      context: .
+      dockerfile: ./e2e-docker/Dockerfile.consumer
+    expose: 
+      - 8080
+    command: sh -c "GRPC_VERBOSITY=debug /tmp/bazel-bin/test/e2e/e2e_consumer"
+
+  provider:
+    build:
+      context: .
+      dockerfile: ./e2e-docker/Dockerfile.provider
+    ports:
+      - 8081:8081
+    command: sh -c "GRPC_VERBOSITY=debug /tmp/bazel-bin/test/e2e/e2e_provider"
```

Comment:
Without health checks and services dependencies, you may have flaky test result some times

Reply:
@kezhenxu94  I think that `depends_on` should be required but `healthcheck` is not required because it has functionality to wait connection on SDK. https://github.com/SkyAPM/cpp2sky/blob/main/source/grpc_async_client_impl.cc#L69-L73.
And, docker-compose > 3 doesn't support `depends_on.condition`. So it will be more complex because it requires strange script to wait. https://docs.docker.com/compose/startup-order/
If you have any idea to resolve this problem more efficient way, let me know about it.